components/Activitylist/ActivityCard.js

```diff
@@ -1,23 +1,25 @@
-import Image from "next/image";
 import Link from "next/link";
 
-export default function ActivityCard({ _id, title, area, country, description }) {
+export default function ActivityCard({
+  _id,
+  title,
+  country,
+  categories,
+  area,
+  imageUrl
+}) {
   return (
     <li>
       <Link href={`/activities/${_id}`}>
-       <img
-              src="https://images.unsplash.com/photo-1551698618-1dfe5d97d256?w=300&h=300&fit=crop&crop=center"
-              alt={title}
-              height={300}
-              width={300}
-            />
-      <h3>{title}</h3>
+        <img src={imageUrl} alt={title} height={300} width={300} />
       </Link>
-      <p>{description}</p>
+      <h3>{title}</h3>
       <ul>
         <li>{area}</li>
         <li>{country}</li>
       </ul>
```

Comment:
here you would not use a ul, since you display two different types of information area and country